operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -399,3 +399,9 @@ Return if we should create a SecurityContextConstraints. Takes into account user
 {{- define "mimir.rbac.useSecurityContextConstraints" -}}
 {{- and .Values.rbac.create (eq .Values.rbac.type "scc") -}}
 {{- end -}}
+
+{{- define  "mimir.remoteWriteEndpoint.inCluster" -}}
```

Comment:
this logic seems very similar to `mimir.gatewayUrl` above, I'd suggest to remove it or reuse gatewayUrl inside